bokehjs/src/coffee/api/plugin.coffee

```diff
@@ -0,0 +1,27 @@
+
+define [
+  "underscore"
+  "jquery"
+  "common/logging"
+  "./figure"
+  "./helpers"
+], (_, $, Logging, figure, helpers) ->
+
+  logger = Logging.logger
+  show = helpers.show
+
+  _api = {
+    "figure": figure
+  }
+
+  $.fn.bokeh = (type, args) ->
```

Comment:
There is an in-bokeh well defined API, it's the three lines that comprise the definition of the plugin. If you want to split the plugin out, It think that's great, in fact splitting off `bokehjs-api.js` into a separate package was the first thing I was going to ask for your help will after 0.8.